core/src/main/java/org/apache/carbondata/core/mutate/CarbonUpdateUtil.java

```diff
@@ -80,18 +80,18 @@ public static String getSegmentWithBlockFromTID(String Tid) {
    * Returns block path from tuple id
    *
    * @param tid
-   * @param factPath
+   * @param tablePath
```

Comment:
Please add isSegmentFile and related describe

Reply:
ok